cmd/osbuild-jobsite-builder/main.go

```diff
@@ -253,91 +228,82 @@ func (builder *Builder) HandleBuild(w http.ResponseWriter, r *http.Request) erro
 	envs := os.Environ()
 	envs = append(envs, buildRequest.Environments...)
 
-	builder.Build = &BackgroundProcess{}
-	builder.Build.Process = exec.Command(
+	b.Build = &BackgroundProcess{}
+	b.Build.Process = exec.Command(
 		"/usr/bin/osbuild",
```

Comment:
(nipitck) if we just use "osbuild" and rely on PATH this will be easier testable by providing a mock binary in a test controlled PATH. It should have no security implications because if someone overrides PATH on our machine we have bigger problems. OTOH there are other ways to test this so fine to ignore.

Reply:
It's definitely a habit of mine to always specify full paths to prevent any `PATH` shenanigans. I'm going to leave it in because as you said there are other ways to test this or we can alternatively make it part of some configuration that can be set by the test cases.